AirCasting/Utils/Strings.swift

```diff
@@ -570,4 +570,21 @@ struct Strings {
     enum InAppAlerts {
         static let assertError: String = "Unsupported button count! For SwiftUI implementation max of 2 buttons is supported"
     }
+    
+    enum AddNoteView {
+        static let title: String = "Add a note"
+        static let description: String = "Your note will be timestamped and displayed on the AirCasting map"
+        static let placeholder: String = "Note"
+        static let continueButton: String = "Add a note"
+        static let cancelButton: String = "Cancel"
```

Comment:
I'm not sure if that's a good idea because it's breaking a "rule" that we have separate enum for each view. 
But!
Maybe we could add an Enum for all Buttons? We would avoid having `cancel`, `continue` or other typical buttons repeated so many times. 

Just an idea to discuss :D

Reply:
Great idea, what would you say to make it in the new PR?

Reply:
Sure! It's not about your code and we can do it as small refactor in the future. 
Anyways, I would ask Ania&Paweł for their opinion before doing it :)

Reply:
I think we should extract common strings to a `Common` enum. So we have all universal strings there like:

```Swift
enum Commons {
    static let cancel: String = "Cancel"
    static let ok: String = "Ok"
    // etc.
}
```

Reply:
Wil do in new PR then, Thanks!